src/chain/chaininterface.rs

```diff
@@ -109,6 +109,11 @@ pub trait FeeEstimator: Sync + Send {
 	///  * satoshis-per-byte * 250
 	///  * ceil(satoshis-per-kbyte / 4)
 	fn get_est_sat_per_1000_weight(&self, confirmation_target: ConfirmationTarget) -> u64;
+	/// Gets satoshis of minimum relay fee required per 1000 Weight-Units.
```

Comment:
Hardcoding 4000sat for 1000 weight somewhere ?

Reply:
Yea. Why not? It's not gonna change any time soon, I don't think, or if it does it'll go down.

Reply:
Also, hardcoding it (instead of reading it from the fuzz input) should remove the full_stack_target changes, which is nice since it saves a bunch of effort regenerating them.